client/src/components/BoardGameForm.js

```diff
@@ -48,21 +48,30 @@ const BoardGameForm = (props) => {
         throw(error)
       } else {
         const responseBody = await response.json()
-        // do something after the form is submitted
+        // debugger
+        const newGameId = responseBody.id
+        setRedirect({
+          status: true,
+          redirectId: newGameId
+        })
```

Comment:
Feel free to just redirect to the index page if you'd rather -- that's what they'll need to know how to do for the System Check :)

Reply:
In fact, I might suggest this just to prevent confusion with the redirect pattern. You can show them this fanciness if you have extra time at the end perhaps?